pinc/page_tally.inc

```diff
@@ -32,19 +32,21 @@ foreach ( $Round_for_round_id_ as $round )
 
 // -----------------------------------------------------------------------------
 
-// The following variables assume that the site has been configured with an
-// "Entry-Level Round" (ELR): a single round where new users start out and
-// become familiar with the site.  The page-tallies from this round are
-// distinguished from others in that they are (deemed to be) a fair indicator
-// of the user's experience.
-
-$users_ELR_page_tallyboard = new TallyBoard( $ELR_round->id, 'U' );
-$teams_ELR_page_tallyboard = new TallyBoard( $ELR_round->id, 'T' );
```

Comment:
`stats/includes/team.inc` has a reference to the (ex-)global `$teams_ELR_page_tallyboard`. It will need to be handled similarly to `stats/includes/member.inc`

Reply:
Good catch - thanks for finding that!